cardscan-base/src/main/java/com/getbouncer/cardscan/base/MachineLearningThread.java

```diff
@@ -21,21 +21,24 @@
 import java.util.ArrayList;
 import java.util.LinkedList;
 
-class MachineLearningThread implements Runnable {
-
-    class RunArguments {
-        private final byte[] mFrameBytes;
-        private final Bitmap mBitmap;
-        private final OnScanListener mScanListener;
-        private final OnObjectListener mObjectListener;
-        private final Context mContext;
-        private final int mWidth;
-        private final int mHeight;
-        private final int mFormat;
-        private final int mSensorOrientation;
-        private final float mRoiCenterYRatio;
-        private final boolean mIsOcr;
-        private final File mObjectDetectFile;
+public class MachineLearningThread implements Runnable {
+
+    protected class RunArguments {
+        public final byte[] mFrameBytes;
+        public final Bitmap mBitmap;
+        public final OnScanListener mScanListener;
+        public final OnObjectListener mObjectListener;
+        public final OnUXModelListener mUXModelListener;
+        public final Context mContext;
+        public final int mWidth;
+        public final int mHeight;
+        public final int mFormat;
+        public final int mSensorOrientation;
+        public final float mRoiCenterYRatio;
+        public final boolean mIsOcr;
+        public final File mObjectDetectFile;
+        public final boolean mRunOcr;
```

Comment:
It's confusing that we have `mIsOcr` and `mRunOcr`. Can we make these more distinct or clear as to what they do?

Reply:
yeah, hmm. i *could* just overload mIsOcr, but essentially mIsOcr is used by every other flow to tell MachineLearningThread to run OCR (as opposed to object detector), and mRunOcr is used to tell us to run the OCR model on top of whatever else it's running (in the UXMLT). Renaming it for now